ecs-agent/acs/session/attach_resource_responder_test.go

```diff
@@ -109,6 +113,20 @@ func TestValidateAttachResourceMessage(t *testing.T) {
 	_, err = validateAttachResourceMessage(&confirmAttachmentMessageCopy)
 
 	require.Error(t, err)
+
+	confirmAttachmentMessageCopy = *testConfirmAttachmentMessage
+	confirmAttachmentMessageCopy.Attachment = nil
```

Comment:
Is this check from the one in here ?
https://github.com/aws/amazon-ecs-agent/pull/3826/files#diff-4063b0cee316c7ff8d0ea85cbf96c5a7d8ea726b84bd0548bb314c45cb6f131aR90

Reply:
I must have accidentally copied that over. Good catch! I'll remove this duplicated check.